dags/main_summary.py

```diff
@@ -420,6 +420,34 @@
     uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
     dag=dag)
 
+sql_clients_daily = bigquery_etl_query(
+    task_id="sql_clients_daily",
+    destination_table="clients_daily_v6",
+    project_id="moz-fx-data-shared-prod",
+    dataset_id="telemetry_derived",
+    owner="relud@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
+    start_date=datetime(2019, 11, 5),
+    dag=dag)
+
+sql_clients_daily_export = SubDagOperator(
+    subdag=export_to_parquet(
+        table="moz-fx-data-shared-prod:telemetry_derived.clients_daily_v6",
+        destination_table="sql_clients_daily_v6",
+        arguments=[
+            "--filter=submission_date = DATE '{{ds}}'",
+            "--where=submission_date = DATE '{{ds}}'",
```

Comment:
Are both `filter` and `where` needed here?

Reply:
yes